app/controllers/api/braintree_controller.rb

```diff
@@ -0,0 +1,33 @@
+class Api::BraintreeController < ApplicationController
+  # TODO: replace with something smarter / safer
```

Comment:
I reckon it's fine to merge TODOs into my branch but we should plan TODO them all before we make it to development